packages/manager/modules/pci/src/projects/project/public-gateways/delete/delete.controller.js

```diff
@@ -0,0 +1,48 @@
+export default class PciPublicGatewaysDeleteController {
+  /* @ngInject */
+  constructor($translate, PciPublicGatewaysService) {
+    this.$translate = $translate;
+    this.PciPublicGatewaysService = PciPublicGatewaysService;
+  }
+
+  $onInit() {
+    this.isDeleting = false;
+    this.trackPublicGateways('public-gateway::delete', 'page');
+  }
+
+  onCancel() {
+    this.trackPublicGateways('public-gateway::delete::cancel');
```

Comment:
You use 'public-gateway::delete::' twice. Maybe make a constant.

Reply:
updated as suggested.